autoprecompiles/src/lib.rs

```diff
@@ -245,6 +241,7 @@ impl<T: FieldElement> PcLookupBusInteraction<T> {
 }
 
 /// A configuration of a VM in which execution is happening.
+#[derive(Clone)]
```

Comment:
Before this PR, we create many new instances of the config by cloning the fields in the constructor. After this PR, we create a single instance and clone it where needed (each apc generation call)